### DISCUSSION TITLE
Failed to set code signing settings for ios

### DISCUSSION THREAD

**Discussion by Skyrac** [codemagic-ci-cd/codemagic-docs#1156] [Q&A] [answered]
I pretty much followed this video: https://docs.codemagic.io/yaml-code-signing/signing-ios/ in setting up.

Did I do something wrong? :(

![image](https://user-images.githubusercontent.com/35764417/201912106-a0e9acc3-8098-4a21-b24c-71d0322f61f8.png)

**Comment by himesh-cm** [comment]
Hi @Skyrac, Can you please share the build id?

**Reply by Skyrac** [reply]
build id: 63737c2a7cf70bf10b917bdb

**Reply by himesh-cm** [reply]
Checking

**Reply by himesh-cm** [reply]
Hi @Skyrac, can you please set up the order of scripts as mentioned [here](https://docs.codemagic.io/yaml-quick-start/building-a-flutter-app/#conclusion)  in the iOS section and re-run the build?

**Reply by Skyrac** [reply]
Hi @himesh-cm 

Done with several tests. Also tried the workflow editor but build fails with same error.
Signing etc. is automatically fetched by iOS API Key

Build Id: 6374dc9122caac6a7fff2cd6

**Reply by himesh-cm** [reply]
Checking

**Reply by himesh-cm** [reply]
Hi @Skyrac, There is no .xcodeproj file in your iOS directory.
Did you try running the app locally? Does it work?
error from the logs
```
Searching for files matching /Users/builder/clone/ios/**/*.xcodeproj
Failed to set code signing settings for ._Runner.xcodeproj
```

Also, please share the directory structure of your iOS folder.

**Reply by Skyrac** [reply]
![image](https://user-images.githubusercontent.com/35764417/202213176-9191c58d-84d0-48a0-a2e8-d394cc63d6c8.png)

One of our devs said it's running with virtual ios devide on his mac

**Reply by himesh-cm** [reply]
I will have a look at it tomorrow and get back to you.

**Comment by icarusdust** [comment, marked as answer]
The very first thing that caught my attention from the screenshot you attached is this part:

`Failed to set code signing settings for ios/._Runner.xcodeproj`

I would expect it to be `ios/Runner.xcodeproj` instead. Checking your second screenshot and there is `._ `indeed under Runner.xcworkspace. 

Possibly, you can get rid of both `._Runner.xcodeproj` and `._Runner.xcworkspace ` as they are unnecessary and do not know why you added them.

**Reply by Skyrac** [reply]
That seems to be the error 👍 
Thank you very much
